tendrl/node_agent/ini2json.py

```diff
@@ -0,0 +1,125 @@
+from ConfigParser import ConfigParser
```

Comment:
Will do as part of another PR later as its referred in gluster-integration as well. Hope you are ok with that :)